src/app/pages/files/files.page.ts

```diff
@@ -12,19 +13,30 @@ import { AuthService } from '../../services/auth.service';
 export class FilesPage implements OnInit {
   private items: Array<FileEntry>;
   private files: FileList;
+  baseURL: string;
 
   constructor(public fileService: FileService,
     public alertCtrl: AlertController,
-    public auth: AuthService) { }
+    public auth: AuthService, openShift: OpenShiftConfigService) {
+    // Note: File Endpoint assume that server returns relative paths to files
+    this.baseURL = new URL(openShift.getServerUrl()).origin;
```

Comment:
Where does the `URL` constructor come from? Is it a global?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/URL/URL

Reply:
https://caniuse.com/#feat=url